src/symbol/glyph_source.js

```diff
@@ -50,22 +55,33 @@ class GlyphSource {
         // the number of pixels the sdf bitmaps are padded by
         const buffer = 3;
 
-        const missing = {};
+        const missingRanges = {};
         let remaining = 0;
 
         const getGlyph = (glyphID) => {
             const range = Math.floor(glyphID / 256);
+            if (this.localIdeographFontFamily &&
+                (isChar['CJK Unified Ideographs'](glyphID) ||
+                 isChar['Hangul Syllables'](glyphID))) {
```

Comment:
Would we be able to support characters in the supplementary planes (#4001), now that we’re sourcing the glyphs from system fonts? Or are we still tied to the BMP due to how the glyphs are stored?